linux.mk

```diff
@@ -31,6 +31,7 @@ export BUILD_DIR := $(CURDIR)/target/release
 ifeq ($(BUILD),dev)
 export BUILD_DIR := $(CURDIR)/target/debug
 endif
+export INPUT_DIR ?= $(CURDIR)/input
```

Comment:
Can this directory be moved inside the nettest directory? Currently it's in the Demikernel root dir so it's hard to associate it with where it belongs.

Reply:
Sure.